docs/index.yml

```
@@ -40,11 +40,11 @@ highlightedContent:
       url: https://developer.microsoft.com/office/blogs/technology/add-ins/
     # Card 6
     - itemType: how-to-guide
-      title: Beginner's guide
+      title: Office Add-ins Beginner's guide
       url: overview/learning-path-beginner.md
     # Card 7
     - itemType: reference
-      title: API reference documentation
+      title: Office JavaScript API reference documentation
       url: https://docs.microsoft.com/javascript/api/overview
```

Should be able to use a relative path here
```suggestion
      url: /javascript/api/overview
```

---

There are a number of docs.microsoft.com links in this file, should I make this same change for all of them? 
For example: https://docs.microsoft.com/office/dev/store/ -> /office/dev/store/

---

Let's confirm that the link still works then update the rest